### PR TITLE
Feature/AW-34 | Teacher - Class management

### PR DESCRIPTION
**Pull request by anhkhoajjo**

### CODE REVIEW

**Comment by anhkhoajjo** [comment]
[Teacher - Class management](https://trello.com/c/okZCK8KF/34-teacher-class-management)

File: src/main/java/com/ptudw/web/web/rest/CourseResource.java [hunk 1]
```diff
@@ -185,6 +185,20 @@ public ResponseEntity<Course> getCourse(@PathVariable Long id) {
...
+     * {@code GET  /courses/:id} : get the "id" course.
+     *
+     * @param id the id of the course to retrieve.
+     * @return the {@link ResponseEntity} with status {@code 200 (OK)} and with body the course, or with status {@code 404 (Not Found)}.
+     */
+    @GetMapping("/courses/joined-course/{userId}")
+    //TODO: get all joined courses by userId
+    public ResponseEntity<Course> getJoinedCoursesByUserId(@PathVariable Long userId) {
+        log.debug("REST request to get all joined courses by userId : {}", userId);
+        Optional<Course> course = courseService.findOne(userId);
```
Comment (huykevin2002) [inline src/main/java/com/ptudw/web/web/rest/CourseResource.java:198]: The method is "getJoinedCoursesByUserId", but I see that it returns only one course, it is okay?

File: src/main/webapp/app/shared/components/class-card/class-card.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export default function ClassCard(prop: IClassCardProp) {
+  const navigate = useNavigate();
+
+  const showDetail = () => {
+    navigate('/teacher/class-management/detail');
+  };
+  const header = (
+    <div className="position-relative">
+      {/* <img alt="Card" src="https://primefaces.org/cdn/primereact/images/usercard.png" /> */}
+      <img className="aw-card-header-image" src="../../../../content/images/title-area-left-background-holiday.jpg" alt="" />
```
Comment (huykevin2002) [inline src/main/webapp/app/shared/components/class-card/class-card.tsx:20]: It must be `src="content/images/title-area-left-background-holiday.jpg"`

**Review by huykevin2002 — CHANGES_REQUESTED** [review]
Huy's Code Review

**Review by huykevin2002 — APPROVED** [review]

**Pull request merged**